### ISSUE TITLE
Change default hash?

### ISSUE THREAD

**Issue by willow-ahrens** [beacon-biosignals/StableHashTraits.jl#19]
Hello! I greatly appreciate this package! would it be possible to change the default hash in a given context? So many datatypes don't have `write` defined (nor does it really make sense to define it). A better default for me would be `UseQualifiedName(UseProperties())`, and only using `write` on isbits types. Similarly, defaulting to `write` sometimes leads to collisions when two objects have different types but the same binary representation, and I'd like to include the type of the object in the hash by default. Any tips?

**Comment by palday** [comment]
https://github.com/beacon-biosignals/StableHashTraits.jl#avoiding-type-piracy has some tips for changing the default hash for types in your context. Something like

```julia
struct MyContext end
StableHashTraits.hash_method(::Any, ::MyContext) = UseProperties(:ByOrder)
stable_hash(x; context=MyContext())
```

**Comment by haberdashPI** [comment]
We're considering making this change for the default context in #22

**Comment by haberdashPI** [comment]
Closed by #24
